src/mmrelay/db_utils.py

```
@@ -566,73 +573,78 @@ def _initialize(cursor: sqlite3.Cursor) -> None:
         # Migrate legacy message_map schema where meshtastic_id used INTEGER affinity.
-        cursor.execute("PRAGMA table_info(message_map)")
+        cursor.execute(f"PRAGMA table_info({MESSAGE_MAP_TABLE})")
         columns = cursor.fetchall()
         column_map = {column[1]: column for column in columns}
-        meshtastic_column = column_map.get("meshtastic_id")
-        meshnet_column = column_map.get("meshtastic_meshnet")
+        meshtastic_column = column_map.get(_col_id)
+        meshnet_column = column_map.get(_col_mesh)
         cursor.execute(
-            "SELECT name FROM sqlite_master WHERE type='table' AND name='message_map_legacy'"
+            f"SELECT name FROM sqlite_master WHERE type='table' AND name='{_legacy_table}'"  # nosec B608
```

Ruff (0.15.7)

[error] Possible SQL injection vector through string-based query construction

(S608)